DashAI/back/dataloaders/classes/dashai_dataset.py

```diff
@@ -120,6 +121,49 @@ def change_columns_type(self, column_types: Dict[str, str]):
         dataset = self.cast(new_features)
         return dataset
 
+    def sample(
+        self,
+        n: int = 1,
+        method: Literal["head", "tail", "random"] = "head",
+        seed: int | None = None,
+    ):
```

Comment:
Question: what kind of object returns, a python dict, dashai dataset or huggingface dataset?

Please add a return type to make this explicit